### PR TITLE
app-editors/remarkable: enable py3.12, drop old

### PR DESCRIPTION
**Pull request by madmartin**
app-editors/remarkable: enable py3.12
    app-editors/remarkable: drop 1.9.0_pre20210320-r2
    
    Signed-off-by: Martin Dummer <martin.dummer@gmx.net>

### CODE REVIEW

File: app-editors/remarkable/remarkable-1.9.0_pre20210320-r3.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
```
Comment (mgorny) [inline app-editors/remarkable/remarkable-1.9.0_pre20210320-r3.ebuild:6]: Remove 3.9, it's no longer supported and causes pkgcheck warnings.

Comment (madmartin) [reply app-editors/remarkable/remarkable-1.9.0_pre20210320-r3.ebuild:6]: I'll remove it, but pkgcheck 0.10.29 does not give a python related warning.

```
martin@bln9716cm ~/swbuild/gentoo-tree/gentoo/app-editors/remarkable $ pkgcheck scan --net
app-editors/remarkable
  UnstableOnly: for arches: [ amd64, arm64, x86 ], all versions are unstable: [ 1.9.0_pre20210320-r3 ]
```

Comment (mgorny) [reply app-editors/remarkable/remarkable-1.9.0_pre20210320-r3.ebuild]: You need to run it with `--commits`.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**